packages/sampler/stories/default/ViewManager.js

```diff
@@ -42,8 +46,8 @@ const ViewManagerLayout = (props) => {
 		setSelected(ev.selected);
 	}, [setSelected]);
 
-	const selectedEnd = select('end', Object.keys(prop.end), ViewManagerConfig, 'index');
-	const selectedStart = select('start', Object.keys(prop.start), ViewManagerConfig, 'index');
+	// eslint-disable-next-line enact/prop-types
+	const {selectedEnd, selectedStart} = props;
```

Comment:
I'm seeing below console warnings. Let's destructure these and `...rest` and pass `rest` instead of `props`.
```
Warning: React does not recognize the `selectedEnd` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `selectedend` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    at div
```